schema-source/ASHRAE229_extra.schema.yaml

```diff
@@ -4318,15 +4318,15 @@ FluidLoopDesignAndControl:
       AppG Used By TCDs: True
       AppG P_RMD Equals U_RMD: True
       AppG B_RMD Equals P_RMD: False
-      AppG B_RMDs Same: "unknown"
+      AppG B_RMDs Same: True
```

Comment:
False because CHW system could be primary-secondary for one rotation and primary only for another.

Reply:
This would not be tested under the `id` property, but rather under the RulesetModelDescription `fluid_loops` and FluidLoop `child_loops`

Reply:
Stay on the side of allowing things to pass if it might be correct or could be interpreted as correct.